docs/4.4/api-reference.md

```diff
@@ -818,14 +820,99 @@ if err := client.DeleteAccessRequest(ctx, accessReqID); err != nil {
+RotationPhaseUpdateServers = "update_servers"
+// RotationPhaseRollback means that the rotation is rolling back to the old certificate authority, may occur in the case of a rotation failure.
+RotationPhaseRollback = "rollback"
+// The rotation will end in Standby after the rotation completes
+```
+
+You can also rotate certificates with a custom schedule. The default schedule gives the `Init`, `UpdateClients`, and `UpdateServers` phases each 1/3 of the total grace period to complete. You may want to extend the length of one phase in the schedule without having a ridiculously long grace period, since long grace periods present a possible vulnerability
+
+```go
+// This will automate your CA rotation with the custom schedule. Use with caution, each phase should have times extra time to ensure they complete in less than optimal situations.
+clock := clockwork.NewRealClock()
+if err := client.RotateCertAuthority(auth.RotateRequest{
+  Mode: services.RotationModeAuto,
+  Schedule: &services.RotationSchedule{
+    // 1 hour for Init
+    UpdateClients: clock.Now().UTC().Add(time.Hour).UTC(),
+    // 4 hours for UpdateClients
+    UpdateServers: clock.Now().UTC().Add(time.Hour * 5).UTC(),
+    // 2 hours for UpdateServers
+    Standby: clock.Now().UTC().Add(time.Hour * 7).UTC(),
+  },
+}); err := client.RotateCertAuthority(req); err != nil {
+  return err
+}
+```
+
+The best option is to set up a custom system of auth rotation using `manual` mode, and triggering each phase manually, so that the rotation does not depend on an arbitrary rotation schedule based on an arbitrarily long grace period.
+
+You can set up a custom automated solution using `manual` mode where each phase is triggered by the event of it completing. For example, if you are tracking a cluster's servers and know when they have all updated to the new certificates, you can trigger the next phase. This is both quicker, and more error proof in cases where the servers go offline and don't have time to update their certificates.
+
+You can also catch errors in the rotation and trigger the `Rollback` phase to try again.
 
 ```go
+// You can run this RPC to start the Update Servers phase
+if err := client.RotateCertAuthority(auth.RotateRequest{
+  Mode:        services.RotationModeManual,
+  TargetPhase: services.RotationPhaseUpdateServers,
+}); err != nil {
+  return err
+}
+```
+
+### Retrieve Certificate Authority
+
+If you need to access specific information on your CA in a program, you can use the following RPC to retrieve the CA and view its keys, certificates, and more. This can also be done using `tctl auth export`.
+
+```go
+// retrieve the cluster's Certificate Authority for Hosts
 ca, err := client.GetCertAuthority(services.CertAuthID{
   DomainName: clusterName,
   Type:       services.HostCA,
 }, false)
 if err != nil {
   return err
 }
-```
```

Comment:
I would also show how to unmarshal the certificate and return a https://golang.org/pkg/crypto/x509/#Certificate